cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,63 @@
+import PageObject from '../PageObject';
+
+class SettingsPageObject extends PageObject {
+  url = '/settings';
+
+  get usernameField() {
+    return cy.getByDataCy('settings-username-field');
+  }
+
+  editUsername() {
+    this.usernameField.clear().type(username);
```

Comment:
The `editUsername` method is missing the `username` parameter. You should pass `username` as an argument to this method to ensure it works correctly.